webpack.config.js

```diff
@@ -8,30 +7,33 @@ module.exports = {
     contentScript: "./src/content_script.js",
     backgroundScript: "./src/background_script.js",
     panel: "./src/panel.js",
-    panelApp: "./src/panel-app.js"
+    panelApp: "./src/panel-app.js",
   },
   plugins: [
-    new CleanWebpackPlugin({
```

Comment:
Doesn't seem to work in webpack 5, even the new clean-webpack-plugin@4 which is supposed to support webpack 5